Babylon/commands/api/scenarioruns/service/api.py

```diff
@@ -9,22 +9,18 @@
 
 class ScenarioRunService:
 
-    def __init__(self, state: dict, azure_token: str, spec: Optional[dict] = None):
+    def __init__(self, azure_token: str, state: dict, spec: Optional[dict] = None):
```

Comment:
finally, this service doesn't need a `spec` argument, could you please remove it ? :slightly_smiling_face: